book/src/dev/zebra-dependencies-for-audit.md

```diff
@@ -0,0 +1,172 @@
+| [zebra-chain](https://github.com/ZcashFoundation/zebra/tree/main/zebra-chain) | [1.0.0-beta.16](https://github.com/ZcashFoundation/zebra/tree/v1.0.0-rc.0/zebra-chain) |
+| [zebra-consensus](https://github.com/ZcashFoundation/zebra/tree/main/zebra-consensus) | [1.0.0-beta.16](https://github.com/ZcashFoundation/zebra/tree/v1.0.0-rc.0/zebra-consensus) |
+| [zebra-network](https://github.com/ZcashFoundation/zebra/tree/main/zebra-network) | [1.0.0-beta.16](https://github.com/ZcashFoundation/zebra/tree/v1.0.0-rc.0/zebra-network) |
+| [zebra-node-services](https://github.com/ZcashFoundation/zebra/tree/main/zebra-node-services) | [1.0.0-beta.16](https://github.com/ZcashFoundation/zebra/tree/v1.0.0-rc.0/zebra-node-services)  |
+| [zebra-rpc](https://github.com/ZcashFoundation/zebra/tree/main/zebra-rpc) | [1.0.0-beta.16](https://github.com/ZcashFoundation/zebra/tree/v1.0.0-rc.0/zebra-rpc) |
+| [zebra-script](https://github.com/ZcashFoundation/zebra/tree/main/zebra-script) | [1.0.0-beta.16](https://github.com/ZcashFoundation/zebra/tree/v1.0.0-rc.0/zebra-script) |
+| [zebra-state](https://github.com/ZcashFoundation/zebra/tree/main/zebra-state) | [1.0.0-beta.16](https://github.com/ZcashFoundation/zebra/tree/v1.0.0-rc.0/zebra-state) |
+| [zebrad](https://github.com/ZcashFoundation/zebra/tree/main/zebrad) | [1.0.0-rc.0](https://github.com/ZcashFoundation/zebra/tree/v1.0.0-rc.0/zebrad) |
+
+### Zcash/ZF dependencies
+
+| Name | Version | Notes
+|------| ------- | -----
+| [ed25519-zebra](https://github.com/ZcashFoundation/ed25519-zebra) | [3.0.0](https://github.com/ZcashFoundation/ed25519-zebra/releases/tag/3.0.0) |
+
+---
+## Partial Audit 
+
+### Crates
+
+| Name | Version | Notes
+|------| ------- | -----
+| [zebra-utils](https://github.com/ZcashFoundation/zebra/tree/main/zebra-utils) | [1.0.0-beta.16](https://github.com/ZcashFoundation/zebra/tree/v1.0.0-rc.0/zebra-utils) | <i>Only [zebra-checkpoints](https://github.com/ZcashFoundation/zebra/tree/v1.0.0-rc.0/zebra-utils/src/bin/zebra-checkpoints) utility needs to be audited.</i>
+
+### Zcash/ZF dependencies
+
+| Name | Version | Audited | Notes
+|------| --------|-------- | -----
+| [zcash_proofs](https://github.com/zcash/librustzcash.git?rev=ca84123038b64f0f4aa5615f7cf224fbf7ece766#ca841230) | [0.7.1](https://github.com/zcash/librustzcash.git?rev=ca84123038b64f0f4aa5615f7cf224fbf7ece766#ca841230/releases/tag/v0.7.1) | [qedit](https://hackmd.io/@qedit/zcash-nu5-audit) | <i>Only the changes in [PR #666](https://github.com/zcash/librustzcash/pull/666) to [downloadreader.rs](https://github.com/zcash/librustzcash/blob/75b403686261a055b3061e7d168b3a4c86378ed4/zcash_proofs/src/downloadreader.rs) and [lib.rs](https://github.com/zcash/librustzcash/blob/75b403686261a055b3061e7d168b3a4c86378ed4/zcash_proofs/src/lib.rs) need to be audited. </i>  |
```

Comment:
Between Marek and I, we've pretty much rewritten the Zcash parameter download code. `zcashd` doesn't use this code, and I'm not sure if they got it audited with or without my changes.

So let's just get all the download code reviewed, because Zebra depends on it?

```suggestion
| [zcash_proofs](https://github.com/zcash/librustzcash/tree/zcash_proofs-0.8.0) | [0.8.0](https://github.com/zcash/librustzcash/tree/zcash_proofs-0.8.0) | [qedit](https://hackmd.io/@qedit/zcash-nu5-audit) | <i>Only the proof parameter download code in [downloadreader.rs](https://github.com/zcash/librustzcash/blob/zcash_proofs-0.8.0/zcash_proofs/src/downloadreader.rs), [hashreader.rs](https://github.com/zcash/librustzcash/blob/zcash_proofs-0.8.0/zcash_proofs/src/hashreader.rs), and [lib.rs](https://github.com/zcash/librustzcash/blob/zcash_proofs-0.8.0/zcash_proofs/src/lib.rs) need to be audited.</i>  |
```

Reply:
Indeed, the parameter download code in `downloadreader.rs`, etc. has not been audited.